pom.xml

```diff
@@ -177,5 +177,30 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+
+  <repositories>
```

Comment:
From gitter, sounds like I should remove this before merge, and we'll follow up later and remove from other RC1 projects as well.

Reply:
yes, agree.